Source/Scene/Expression.js

```diff
@@ -1273,5 +1279,12 @@ define([
         }
     };
 
+    function getEvaluateUnaryFunction(call) {
+        var evaluate = unaryFunctions[call];
+        return function(feature) {
+            return evaluate(this._left.evaluate(feature));
+        }
+    }
+
```

Comment:
Place this function closer to the other evaluate functions. Right above `evaluateLiteral` is fine.